.github/workflows/test.yaml

```diff
@@ -35,6 +35,14 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Disable Login
```

Comment:
**question:** Is there a recommended way to use this locally to run these same tests? If it is just "add this line to settings.py", could we document that in e.g. a (new) `docs/developer/testing.md`?

Reply:
It is just "add this line to settings.py"... and I have updated the docs. :)